lib/fs.js

```diff
@@ -923,6 +924,26 @@ function stat(path, options = { bigint: false }, callback) {
   binding.stat(pathModule.toNamespacedPath(path), options.bigint, req);
 }
 
+function statfs(path, options = { bigint: false }, callback) {
+  if (typeof options === 'function') {
+    callback = options;
+    options = {};
+  }
+
+  if (typeof callback !== 'function') {
```

Comment:
Can you change this `if` statement to `callback = maybeCallback(callback)` for code reuse and coverage.